src/mixins/helpers.js

```diff
@@ -302,7 +302,7 @@ var helpers = {
         this.slideHandler(nextIndex);
       }
     };
-    if (this.props.autoplay) {
+    if (isInitManually == true || this.props.autoplay) {
```

Comment:
should clearInterval before setting a new one to make sure that all callbacks are accounted for.